modules/policy-identity-access-management.adoc

```diff
@@ -5,9 +5,11 @@
 
 [id="policy-identity-access-management_{context}"]
 = Identity and access management
+Most access by Red Hat site reliability engineering (SRE) teams is done using cluster Operators through automated configuration management.
 
 
-Most access by SRE teams is done by using cluster Operators through automated configuration management.
+=== Subprocessor
+For a list of the available subprocessors, see the link:https://access.redhat.com/articles/5528091[Red Hat Subprocessor List on the Customer Portal].
```

Comment:
Suggest only having "Red Hat Subprocessor List" as the link text, and then moving the "on the..." outside of the link text. Also suggest calling it the "Red Hat Customer Portal"